database/migrations/2020_12_29_025357_create_modules_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateModulesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('modules', function (Blueprint $table) {
```

Comment:
Maybe add column for extra data?

Reply:
Hi,
This is good suggestion. I will add a extra column.

Reply:
Shouldnt that be done on the app level? In my point of view you publish the migration, and modify it to match your needs

Reply:
Thanks @FoksVHox , you are right.
We can setAttribute and getAtrribute by DatabaseModule class.
I think I can add a function like getExtra(). Do you think it necessary? Because in app level, we can extend this class.

Reply:
@lamtranb Personally I dont think it is needed, in the packages itself

Reply:
I wrote a testcase to check support extra